src/pages/app.tsx

```diff
@@ -208,7 +208,7 @@ const PlatformsInstallSnippet: FunctionComponent = () => (
                                         and install it:
                                     </div>
 
-                                    <CodeSnippet code="dpkg -i <file.deb>" className="-ml-2.5" />
+                                    <CodeSnippet code="dpkg -i <file.deb>" className="-ml-2.5 pb-3" />
```

Comment:
From what I can see in the dev tools this is the right way to do it. The code block has a pb-9 which is too much.